doc/api/fs.md

```diff
@@ -3615,14 +3619,33 @@ It is unsafe to use `fs.writeFile()` multiple times on the same file without
 waiting for the callback. For this scenario, [`fs.createWriteStream()`][] is
 recommended.
 
-### File Descriptors
-1. Any specified file descriptor has to support writing.
```

Comment:
Are 1 and 2 skipped intentionally?

Reply:
Yes, now, the difference from fs.write() is described, rather than the previous more complicated way of describing the fd behaviour.

Reply:
3 was skipped as well -- which makes the entire previous list being replaced.